helpers/compile/plugins/fill-plugin/fillers/buffer-small.ts

```diff
@@ -0,0 +1,504 @@
+/* eslint-disable @typescript-eslint/no-unsafe-declaration-merging */
+export interface Buffer {
```

Comment:
The types are manually set here because they are added dynamically and not hand written.